### PR TITLE
Update documentation to maistra 0.3.0: MAISTRA-78

### PR DESCRIPTION
**Pull request by knrc**

### CODE REVIEW

**Comment by knrc** [comment]
@JStickler Can you or Elise please take a look at this?

File: servicemesh-install/topics/install.adoc [hunk 1]
```diff
@@ -230,16 +230,11 @@ The OpenShift Master Public URL must be configured to match the public URL of yo
...
 The previous commands create a new deployment within the `istio-operator` project and run the operator responsible for managing the state of the {ProductName} control plane through the custom resource.
 
-. To verify that the operator is installed correctly, execute the following command:
-+
-```
-$ oc get pods -n istio-operator
-```
-+
-. You can access the logs from the `istio-operator` pod with the following command, replacing `<pod name>` with the name of the pod you discovered.
+. To verify that the operator is installed correctly, access the logs from the operator pod using the following command
```
Comment (kalexand-rh) [inline servicemesh-install/topics/install.adoc]: s/using the following command/by running the following command:

**Review by kalexand-rh — COMMENTED** [review]
@knrc, if you're bumping all the required versions from 3.10 to 3.11, please open a PR against the 3.10 branch to remove the docs from the collection.

**Comment by knrc** [comment]
@kalexand-rh The 3.10 content would still be valid for 3.10, do you normally delete whole revisions in documentation?

File: servicemesh-install/topics/install.adoc (hunk 1 above)
Comment (knrc) [reply servicemesh-install/topics/install.adoc]: I'll make this change and repush

**Comment by kalexand-rh** [comment]
@knrc, if 0.2.0 is available on 3.10 and 0.3.0 is only available on 3.11, then we can keep the content in 3.10 and update only the 3.11 branch.

File: servicemesh-install/topics/install.adoc (hunk 1 above)
Comment (knrc) [reply servicemesh-install/topics/install.adoc]: Done and pushed

**Comment by knrc** [comment]
@kalexand-rh +1, sounds good to me

**Review by geekspertise — COMMENTED** [review]
I read through your changes @knrc and they look fine to me.

**Comment by JStickler** [comment]
@knrc a lot of this is duplicated effort.  I've already submitted most of these changes (except for removing 3.10 references, which I'm updating now) in #12578.

**Comment by knrc** [comment]
@JStickler I'm confused, I thought we were supposed to be submitting docs for you to review.  I didn't even know you had submitted any PR so have definitely not reviewed it.

**Comment by JStickler** [comment]
@knrc we need to figure out our process I guess.

**Comment by knrc** [comment]
Closing this in favour of https://github.com/openshift/openshift-docs/pull/12578

**Pull request closed without merging**